site/content/en/docs/Integration Patterns/matchmaker-registration.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Matchmaker requires game server process registration"
+linkTitle: "Matchmaker registration"
+date: 2021-07-27
+weight: 20
+description: >
+    A scenario in which a Matchmaker requires a game server process to register themselves with the matchmaker, and the
+    matchmaker decides which `GameServer` players are sent to.
+---
+
+In this scenario, the `GameServer` process will need to self Allocate when informed by the matchmaker that players 
+are being sent to them.
+
+![Reserved Lifecycle Sequence Diagram](../../../diagrams/gameserver-reserved.puml.png)
+
+{{< alert title="Warning" color="warning">}}
+This does relinquish control over how `GameServers` are packed across the cluster to the external matchmaker. It is likely
+it will not do as good a job at packing and scaling as Agones.
+{{< /alert >}}
+gameserver-reserved.puml.png
```

Comment:
This line seems extraneous (and is rendered in the page).

Reply:
Looks like I left a bad copy/paste! Fixed 👍🏻